bicepconfig.json

```diff
@@ -8,5 +8,8 @@
 	"extensions": {
 		"radius": "br:biceptypes.azurecr.io/radius:latest",
 		"aws": "br:biceptypes.azurecr.io/aws:latest"
+	},
+	"cloud": {
```

Comment:
I thought this was needed to be removed.. is this something specific to the docs repo?

Reply:
we removed it from the radius repo because it was causing issues when we needed to authenticate to azure to push types to the ACR. for that repo, we added it in workflows where needed instead of in the global config. we don't push types to ACR in this repo so it wouldn't cause the same issues as before